### DISCUSSION TITLE
How to GetTopEntryValue(feildcalculator output result) by pyaedt method

### DISCUSSION THREAD

**Discussion by POWER-TRANSFORMER** [ansys/pyaedt#1613] [Q&A] [answered]
Dear all,
How to GetTopEntryValue(feildcalculator output result) by pyaedt method?

oModule.CalcStack("clear")
    oModule.EnterQty("Energy")
    oModule.EnterVol("AllObjects")
    oModule.CalcOp("RZIntegrate")
    Energy = oModule.GetTopEntryValue("Setup1 : LastAdaptive",
                                      [
                                          "Freq:=", "%d" % Hz, "Phase:=", "0deg"
                                      ])

M2D.ofieldsreporter.CalcStack("clear")
M2D.ofieldsreporter.EnterQty("Energy")
M2D.ofieldsreporter.EnterVol("AllObjects")
M2D.ofieldsreporter.CalcOp("RZIntegrate")
Energy =M2D.ofieldsreporter.?

Thank you very much.

**Comment by Samuelopez-ansys** [comment, marked as answer]
@POWER-TRANSFORMER This method is not implemented yet in PyAEDT, please submit an issue and we will add it to PyAEDT. In the meanwhile, you can still use the same command inside your pyaedt code.
